mmedit/datasets/pipelines/augmentation.py

```diff
@@ -546,6 +733,29 @@ def __repr__(self):
         return self.__class__.__name__ + f'hue_range={self.hue_range}'
 
 
+@PIPELINES.register_module()
+class ColorJitter:
+    """An interface for torch color jitter so that it can be invoked in
+    mmediting pipeline."""
+
+    def __init__(self, **kwargs):
+        self.transform = transforms.ColorJitter(**kwargs)
```

Comment:
Is it possible to do that directly on numpy array? This could (probably) save some time on transforming the data to different types.

Reply:
I have adopted the ColorJitter class in `mmocr/datasets/pipelines/transforms.py`.

Reply:
Okay.